packages/eas-cli/src/vcs/clients/git.ts

```diff
@@ -386,3 +433,51 @@ async function setGitCaseSensitivityAsync(
     });
   }
 }
+
+async function assertEnablingGitCaseSensitivityDoesNotCauseNewUncommittedChangesAsync(
+  cwd: string
+): Promise<void> {
+  // Remember uncommited changes before case sensitivity change
+  // for later comparison so we log to the user only the files
+  // that were marked as changed after the case sensitivity change.
+  const uncommittedChangesBeforeCaseSensitivityChange = await gitStatusAsync({
+    showUntracked: true,
+    cwd,
```

Comment:
we are using `rootPath` as cwd for it here, but previously we used `this.maybeCwdOverride`

does this change matter?

Reply:
`rootPath` is supposed to be root of the Git repository and should take `maybeCwdOverride` into consideration:

<img width="488" alt="Zrzut ekranu 2025-01-30 o 20 01 44" src="https://github.com/user-attachments/assets/4ba0ad68-f1bb-4c06-a3f5-e4b77f159370" />